docs/src/content/docs/pt-br/guides/css-and-tailwind.mdx

```diff
@@ -32,8 +32,8 @@ Customize os estilos aplicados ao seu site Starlight ao fornecer arquivos CSS ad
    		starlight({
    			title: 'Documentação com CSS Customizado',
    			customCss: [
-   				// Caminho relativo ao seu arquivo CSS customizado
-   				'./src/styles/customizado.css',
+   +			// Caminho relativo ao seu arquivo CSS customizado
+   +			'./src/styles/customizado.css',
```

Comment:
Not sure if Prettier would fix this for us, but I think we need another tab here, looking at the rendered output:
<img width="187" alt="code example with these two lines not indented as far as needed" src="https://github.com/withastro/starlight/assets/357379/afcbdfe2-c517-431b-9c6c-dcd2dbe12a44">


```suggestion
   +				// Caminho relativo ao seu arquivo CSS customizado
   +				'./src/styles/customizado.css',
```